src/test/scala/exercise3/WhiteNoiseTest.scala

```diff
@@ -0,0 +1,54 @@
+package exercise3
+
+import exercise2.QueueLike
+import zio.test._
+import zio.test.Assertion._
+import zio.test.environment._
+
+import scala.annotation.tailrec
+
+object WhiteNoiseTest extends DefaultRunnableSpec {
+  @tailrec
+  def checkVolume(queue: QueueLike[Double], volume: Double): Boolean = {
+    if (queue.isEmpty)
+      return true
+
+    val front = queue.front().get
+    if (front < -0.5 * volume || front > 0.5 * volume)
```

Comment:
A cool way to test this is using `TestRandom` and specifically `TestRanom.feedDoubles`

this way random values can be set in advance and verified later. No need to test with 440 elements, could just have 3 or 4 set and verified that way